bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/Scheduler.java

```diff
@@ -168,17 +168,7 @@ public FutureValue scheduleToObjectGroup(Object[] params, Function function, Str
         SchedulerItem item = new SchedulerItem(function, params, future);
         future.strand.schedulerItem = item;
         totalStrands.incrementAndGet();
-        ItemGroup group = objectGroup.get();
-        if (group == null) {
-            group = new ItemGroup(item);
-            objectGroup.set(group);
```

Comment:
For nonisolated methods, we use this method. But we can initialize the object group during scheduler init. This is a minor improvemnt